test/src/main/java/org/apache/accumulo/test/fate/accumulo/AccumuloStoreReadWriteIT.java

```diff
@@ -127,6 +127,31 @@ public void testReadWrite() throws Exception {
     }
   }
 
+  @Test
+  public void testReadWriteTxInfo() throws Exception {
+    final String table = getUniqueNames(1)[0];
+    try (ClientContext client =
+        (ClientContext) Accumulo.newClient().from(getClientProps()).build()) {
+      client.tableOperations().create(table);
+
+      AccumuloStore<TestEnv> store = new AccumuloStore<>(client, table);
+
+      long tid = store.create();
+      FateTxStore<TestEnv> txStore = store.reserve(tid);
+
+      try {
+        // Go through all enum values to verify each TxInfo type will be properly
+        // written and read from the store
+        for (TxInfo txInfo : TxInfo.values()) {
+          txStore.setTransactionInfo(txInfo, "value: " + txInfo.name());
+          assertEquals("value: " + txInfo.name(), txStore.getTransactionInfo(txInfo));
+        }
+      } finally {
+        txStore.delete();
```

Comment:
I was trying to think if there is anything that should be checked after delete, but did not think of anything.